packages/terra-date-picker/src/react-datepicker/time.jsx

```diff
@@ -89,13 +93,13 @@ export default class Time extends React.Component {
     }
 
     return (
-      <div className={`react-datepicker__time-container ${(this.props.todayButton) ? 'react-datepicker__time-container--with-today-button' : ''}`}>
-        <div className="react-datepicker__header react-datepicker__header--time">
-          <div className="react-datepicker-time__header">Time</div>
+      <div className={cx(['react-datepicker-time-container', `${(this.props.todayButton) ? 'react-datepicker-time-container--with-today-button' : ''}`])}>
+        <div className={cx('react-datepicker-header react-datepicker-header--time')}>
```

Comment:
I guess the class names should be in an array like : `['react-datepicker-header', 'react-datepicker-header--time']`